pages.ko/common/cksum.md

```diff
@@ -1,7 +1,7 @@
 # cksum
 
 > 파일의 바이트 개수나 CRC 무결성 검사를 계산합니다.
-> 알립니다, 오래된 UNIX 시스템은 CRC 검사가 다를 수 있습니다.
+> 알립니다: 오래된 UNIX 시스템은 CRC 검사가 다를 수 있습니다.
```

Comment:
> 참고: 이전 UNIX 시스템에서는 CRC 구현이 다를 수 있습니다.

How do you like this translation? In Korean, note is often used as a reference.

Reply:
Not really sure, what you mean here 😅 . I changed `,` to `:` in the description. If the translation isn't ideal feel free to update the file.

Reply:
Ah, yes, I will edit the translation of this part later.